discovery-infra/test_infra/consts/consts.py

```diff
@@ -17,7 +17,7 @@ class OpenshiftVersion(Enum):
 BASE_IMAGE_FOLDER = "/tmp/images"
 IMAGE_NAME = "installer-image.iso"
 STORAGE_PATH = "/var/lib/libvirt/openshift-images"
-SSH_KEY = "ssh_key/key.pub"
+SSH_KEY = "~/.ssh/id_rsa.pub"
```

Comment:
get from os env?

Reply:
not today porfavor

Reply:
I mean, it seems like we're already doing it, but from the makefile
We can do a bit bigger refactoring, but I think we should afterwards